packages/mds-compliance-engine/test_data/fixtures.ts

```diff
@@ -249,15 +250,16 @@ export const HIGH_COUNT_POLICY: CountPolicy = {
   ]
 }
 
-export const ARBITRARY_EVENT_TYPES_POLICY: CountPolicy = {
+export const ARBITRARY_EVENT_TYPES_POLICY: Required<CountPolicy> = {
   policy_id: '6d7a9c7e-853c-4ff7-a86f-e17c06d3bd80',
   name: 'Very Low Count Limit',
   description: 'Very low count limit',
   start_date: 1552678594428,
   end_date: null,
   prev_policies: null,
   provider_ids: [],
-  publish_date: null,
```

Comment:
the test that uses this fixture _assumed_ that the policy was active, because the criteria for that was just based on start_date/end_date before, and that was _wrong_. 
Now, a Policy is only 'active' if it has been published, so I'm trying to be consistent when adding a `status` value, not have each Policy example filled with garbage.